src/app/public/modules/repeater/repeater-adapter.service.ts

```diff
@@ -11,7 +16,29 @@ import {
 export class SkyRepeaterAdapterService {
   private host: ElementRef;
 
-  constructor(private repeaterService: SkyRepeaterService) {}
+  constructor(
+    private repeaterService: SkyRepeaterService,
+    private skyAdapterService: SkyCoreAdapterService
+  ) {}
+
+  public getFocusableChildren(element: HTMLElement, options: SkyFocusableChildrenOptions): HTMLElement[] {
```

Comment:
The `element` parameter should be of type `ElementRef` so that the caller doesn't need to "know" what type the adapter needs. The adapter should be responsible for deriving the type that it requires from the generic `ElemenRef` (in this case, the `nativeElement`).
```
public getFocusableChildren(elementRef: ElementRef, options?: SkyFocusableChildrenOptions): HTMLElement[] {
```

Reply:
Good idea. Done.